ui/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@chakra-ui/icons": "^1.0.15",
-    "@chakra-ui/react": "^1.6.5",
```

Comment:
hmm will this be a breaking change for the rest of our site... not sure if this might break some of the things that our site uses.

can you verify that none of the other functionalities in our site break because of this lol

Reply:
from what i've tested, nothing has been broken...